data_utils.py

```diff
@@ -13,7 +14,8 @@
 from torchtext.vocab import Vocab
 from torchtext.data.utils import get_tokenizer
 
-from utils import log
+
+logging.getLogger().setLevel(logging.INFO)
```

Comment:
Do we need this line if we have set logging. basicConfig() ?

Reply:
No, `logging.basicConfig()`'s fine.